mw-webapp/src/router/Router.tsx

```diff
@@ -1,40 +1,48 @@
 import {Route, Routes} from "react-router-dom";
 import {pages} from "src/router/pages";
+import {ProtectedRouterLayer} from "src/router/ProtectedRouter";
 
 /**
  * Router
  */
 export const Router = () => {
+
   return (
     <Routes>
       <Route
         path={pages.allWays.path}
         element={pages.allWays.element}
       />
       <Route
-        path={pages.user.path(":uuid")}
-        element={pages.user.element}
-      />
-      <Route
-        path={pages.way.path(":uuid")}
-        element={pages.way.element}
-      />
-      <Route
-        path={pages.allUsers.path}
-        element={pages.allUsers.element}
-      />
-      <Route
-        path={pages.userProfile.path(":uuid")}
-        element={pages.userProfile.element}
+        path={pages.page404.path}
+        element={pages.page404.element}
       />
       <Route
         path={pages.aboutProject.path}
         element={pages.aboutProject.element}
       />
       <Route
-        path={pages.page404.path}
-        element={pages.page404.element}
+        path={pages.redirect.path}
+        element={pages.redirect.element}
       />
+      <Route element={<ProtectedRouterLayer />}>
+        <Route
```

Comment:
This route UserPage shouldnot be inside protected route because its normal if I will want to see on userPage even if Im not login

Reply:
done